testing.md

```diff
@@ -0,0 +1,37 @@
+# Manual Testing (for Native Query)
+
+At this time, Native Query must be tested manually by reviewers.
+
+The following query is sufficient to test Native Query as it exercises mutliple areas of translation.
+
+1. Ensure you have a local ADF running.
+- Navigate to the `mongo-odbc-driver` repository locally.
```

Comment:
In the past we've tried to keep testing resources in the same repo instead of relying on one of our others.  Since we have a run_adf.sh script in this mongo-powerbi-connector repo, how about we use one of the collections that we're already using here like [sales](https://github.com/mongodb/mongo-powerbi-connector/blob/main/resources/integration_test/testdata/sales.json).  Or alternatively add a few rows of the grades and class collections from the mongo-odbc-driver integration tests. 
For loading you can use mongoimport, add the steps:
```
          MONGO_TOOLS_VERSION=100.6.1
          curl -L -o mongodb-tools.zip \
            https://fastdl.mongodb.org/tools/db/mongodb-database-tools-windows-x86_64-$MONGO_TOOLS_VERSION.zip
          unzip -jn mongodb-tools.zip
          chmod +x mongoimport.exe
 ```

```
          ./mongoimport.exe --uri="mongodb://localhost:28017/supplies" \
            --drop resources/integration_test/testdata/sales.json
```

```
MONGOSH=$(find ./local_adf/ | grep mongo.exe | head -1)

# Replace with the namespace you are adding
$MONGOSH  -u mhuser -p pencil --eval 'db.runCommand({sqlGenerateSchema: 1, sampleNamespaces: ["supplies.sales"], setSchemas: true})' localhost/admin
```

Reply:
Sounds good, I can make the change. In general, what do you think about converging the data we use to test across our different projects?

Reply:
I am okay with not converging the data used for the different projects.  I think the datasets are usually chosen for testing specific capabilities of the drivers and might not be super useful to have the same dataset for all repos.